packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -87,7 +89,11 @@ function useMediaUploadQueue() {
         state.queue.map(async (item) => {
           const { id, file, state: itemState, resource } = item;
           if (
-            itemState !== 'TRANSCODED' ||
+            ![
+              ITEM_STATUS.TRIMMED,
+              ITEM_STATUS.MUTED,
+              ITEM_STATUS.TRANSCODED,
+            ].includes(itemState) ||
```

Comment:
I am going to leave this as it is for now and follow up with another ticket to remove this as needed.